API/Modules.md

```diff
@@ -14,12 +14,12 @@ The best place to find API documentation for a Tessel module is through the link
 
 ## Quick links
 
-* [See all the modules and find their API documentation](//tessel.io/modules)
-* [Learn how to use modules on the start page](//start.tessel.io/modules)
+* [See all the modules and find their API documentation](https://tessel.io/modules)
+* [Learn how to use modules on the start page](http://tessel.github.io/t2-start/)
```

Comment:
good catch, but should also go in a separate PR. Would like to isolate changes in this PR to gitbook, not to link & fact fixing